cms/standard_pages/models.py

```diff
@@ -38,3 +40,76 @@ class InformationPage(BasePage):  # type: ignore[django-manager-missing]
         index.SearchField("summary"),
         index.SearchField("content"),
     ]
+
+
+class IndexPage(BasePage):  # type: ignore[django-manager-missing]
+    template = "templates/pages/index_page.html"
+
+    parent_page_types: ClassVar[list[str]] = ["home.HomePage", "IndexPage", "InformationPage"]
+    subpage_types: ClassVar[list[str]] = ["IndexPage", "InformationPage"]
+
+    description = models.TextField(blank=True)
+    featured_pages = StreamField([("featured_page", RelatedContentBlock())], blank=True)
+    content = RichTextField(features=settings.RICH_TEXT_BASIC, blank=True)
+    related_links = StreamField([("related_link", RelatedContentBlock())], blank=True)
+
+    content_panels: ClassVar[list["Panel"]] = [
+        *BasePage.content_panels,
+        FieldPanel("description"),
+        FieldPanel("featured_pages"),
+        FieldPanel("content"),
+        FieldPanel("related_links"),
+    ]
+
+    search_fields: ClassVar[list[index.SearchField | index.AutocompleteField]] = [
+        *BasePage.search_fields,
+        index.SearchField("description"),
+    ]
+
+    def get_formatted_featured_pages(self) -> list[dict[str, str]]:
```

Comment:
nitpick: this method returns either the featured items or child pages, so we do need a better name (either more descriptive, or generic). What about `get_formatted_items` ?